apps/omg_db/lib/db.ex

```diff
@@ -80,29 +80,30 @@ defmodule OMG.DB do
     GenServer.call(server_name, :last_exiter_eth_height)
   end
 
-  def init do
+  def init(server_name \\ @server_name) do
```

Comment:
Not rather `server_module`?

Reply:
it's both by coincidence, so decided to be consistent (in other functions it is the server's name that makes more sense, actually)